Venmo/src/main/java/com/braintreepayments/api/venmo/VenmoRequest.kt

```diff
@@ -0,0 +1,123 @@
+ * should be collected and displayed on Venmo pay sheet.
+ *
+ * @property totalAmount Optional - The grand total amount of the transaction that will be
+ * displayed on the pay sheet. The grand total amount of the transaction that will be displayed
+ * on the pay sheet.
+ *
+ * @property subTotalAmount Optional - The subtotal amount of the transaction, excluding taxes,
+ * discounts, and shipping. The subtotal amount of the transaction, excluding taxes, discounts,
+ * and shipping.
+ *
+ * @property discountAmount Optional - The total discount amount applied on the transaction. The
+ * total discount amount applied on the transaction. If this value is set, `totalAmount` must
+ * also be set.
+ *
+ * @property taxAmount Optional - The total tax amount applied to the transaction. The total tax
+ * amount applied to the transaction. If this value is set, `totalAmount` must also be set.
+ *
+ * @property shippingAmount Optional - he shipping amount charged for the transaction. The
+ * shipping amount charged for the transaction. If this value is set, `totalAmount` must also be
+ * set.
+ *
+ * @property isFinalAmount Optional - The boolean value of the flag that signifies whether the purchase
+ * amount is the final amount. Indicates whether the purchase amount is the final amount.
+ * Defaults to false.
+ *
+ * @property isFinalAmountAsString - Whether or not the purchase amount is the final amount as a
+ * string value.
+ */
+@Parcelize
+class VenmoRequest @JvmOverloads constructor(
+    @VenmoPaymentMethodUsage val paymentMethodUsage: Int,
+    var lineItems: ArrayList<VenmoLineItem> = ArrayList(),
+    var shouldVault: Boolean = false,
+    var profileId: String? = null,
+    var displayName: String? = null,
+    var collectCustomerShippingAddress: Boolean = false,
+    var collectCustomerBillingAddress: Boolean = false,
+    var totalAmount: String? = null,
+    var subTotalAmount: String? = null,
+    var discountAmount: String? = null,
+    var taxAmount: String? = null,
+    var shippingAmount: String? = null,
+    var isFinalAmount: Boolean = false
+) : Parcelable {
+
+    val paymentMethodUsageAsString: String?
+        get() = when (paymentMethodUsage) {
+            VenmoPaymentMethodUsage.SINGLE_USE -> "SINGLE_USE"
+            VenmoPaymentMethodUsage.MULTI_USE -> "MULTI_USE"
+            else -> null
+        }
+    /**
+     * Optional: The line items for this transaction. Can include up to 249 line items.
+     *
+     * If this value is set, `totalAmount` must also be set.
+     *
+     * @param lineItems a collection of [VenmoLineItem]
+     */
+    fun setVenmoLineItem(lineItems: Collection<VenmoLineItem>) {
+        this.lineItems.clear()
+        this.lineItems.addAll(lineItems)
+    }
+
+    /**
+     * @return The line items for this transaction. Can include up to 249 line items.
+     */
+    fun getVenmoLineItem(): ArrayList<VenmoLineItem> {
```

Comment:
Do we need this getter function? It looks like the `lineItems` property is public.

Reply:
Our logic around this was kind of complex - I updated to make this an optional parameter, which removes the getter/setter but requires a force unwrap in where it's used in the API but I think it's ok. Let me know what you think

Reply:
Yep, that sounds good to me 👍